tests/certs/scripts/genroot.sh

```diff
@@ -17,6 +17,7 @@ cd $HOME
 
 KEYSIZE=2048
 DURATION=6000
+DIGESTALGO=-sha256
```

Comment:
This bumps the required OpenSSL version for generating the certs to 1.0.1, not that it's a bad thing but perhaps it should be documented (although not entirely sure where..)?

Reply:
Good point, thanks. I figure I can at least add a comment about that fact next to this in the script. I figure that's likely to be seen if you run into problems with this...